Mobile-Workshop/Mobile-Workshop/ProductDetail/ProductDetailView.swift

```diff
@@ -0,0 +1,115 @@
+        VStack(spacing: 0) {
+            headerView
+            
+            ScrollView(.vertical, showsIndicators: false) {
+                VStack(spacing: 20) {
+                    productImageView
+                    
+                    VStack(spacing: 16) {
+                        HStack {
+                            Text("Product Name")
+                                .font(.system(size: 18))
+                                .fontWeight(.bold)
+                            
+                            Spacer()
+                            
+                            HStack(spacing: 0) {
+                                Text("5")
+                                ForEach(0..<5) { _ in
+                                    Image(systemName: "star.fill")
+                                        .foregroundColor(.yellow)
+                                        .font(.system(size: 14))
+                                }
+                            }
+                        }
+                        
+                        HStack {
+                            Text("Category: Home")
+                                .font(.system(size: 12))
+                                .foregroundColor(Color.gray)
+                                .fontWeight(.semibold)
+                            Spacer()
+                        }
+                        Text("Indulge in a heavenly tea experience with our Stargazer’s Tea Set, featuring a constellation-themed teapot and matching teacups. Crafted from fine porcelain, this elegant set will transport you to the cosmos with every sip.")
+                            .multilineTextAlignment(.leading)
+                            .font(.system(size: 16))
+                        
+                        Text("$99.99")
+                            .multilineTextAlignment(.leading)
+                            .font(.system(size: 24))
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                        
+                        Spacer()
+                        
+                        Button(action: {
+                        }) {
+                            Text("Add to cart")
+                                .font(.headline)
+                                .foregroundColor(.white)
+                                .padding()
+                                .frame(width: 250, height: 60)
+                                .background(Color.init(red: 71/255, green: 51/255, blue: 122/255))
+                                .cornerRadius(15.0)
+                        }
+                        
+                    }
+                    .padding()
+                    
+                }
+                .padding(.vertical)
+            }
+            
+            Spacer()
+        }
+        .background(Color.white.edgesIgnoringSafeArea(.all))
+        .navigationBarBackButtonHidden()
+        
+    }
+    
+    
+    var headerView: some View {
```

Comment:
Note: Those can be extracted as structs, be it the same file or a different one. It's a matter of preference but for more complex views it's kind of a must.